### ISSUE TITLE
Refresh does not handle error 404

### ISSUE THREAD

**Issue by johnenrick** [websanova/vue-auth#580]
I am creating a PWA website which should work completely offline. So when there is no internet, I should still be able to visit pages where _meta.auth_ is false. Currently the pages do not load the router-view

**Comment by websanova** [comment]
Well that has nothing to do with being offline or not, it's just a route check.

Your issue seems to be more that you may not have local user data. It's likely you will need to disable `refresh` and set some user data manually if in offline mode.

**Comment by johnenrick** [comment]
Thank you very much for a very quick response. You are right it has nothing to do with the refresh. I have been able to fix the problem with the 404 but I encountered another problem. Its `net::ERR_CONNECTION_REFUSED`.

Here's how I replicated the problem:
1. Log In using the _vue-auth_
2. Turn off the server. In my case the XAMPP
3. Refresh the browser

I am not sure though why the error code changed from 404 to `net::ERR_CONNECTION_REFUSED`. But I verified that _vue-auth_ is preventing my `router.beforeEnter()` from running. I did this by commenting the contents of _jwt-auth.js_.
